docs/plugins/repository-azure.asciidoc

```diff
@@ -162,12 +162,6 @@ The Azure repository supports following settings:
     Specifies the path within container to repository data. Defaults to empty
     (root directory).
 
-`chunk_size`::
```

Comment:
I think we should still keep the docs for chunk_size. Perhaps we can mention that lowering this value can result in much more files in the repo. Still, it's a valid option and should be documented.

Reply:
>  Still, it's a valid option and should be documented.

Is it though? Setting this option has no advantages whatsoever doesn't it? And we had a few cases both internal and external where people mistook this option as some sort of buffer size.

Reply:
> Setting this option has no advantages whatsoever doesn't it?

In the searchable snapshot case, it allows parallel downloads of file chunks. Not saying that that's a good use case, just  that there might be some that we don't know about.

> And we had a few cases both internal and external where people mistook this option as some sort of buffer size.

This could be fixed by improving the documentation :)

I prefer full transparency on this setting (saying what it's default is and why), and actively recommend against tuning it, but not hide it from the docs (as folks might otherwise set it based on old docs / S3  docs and have completely wrong expectations).

Reply:
Makes sense, I pushed https://github.com/elastic/elasticsearch/pull/59279/commits/f45a4f7511bedb26087812f864d9458c7eb08232 :)